### DISCUSSION TITLE
session()->fash(); json error

### DISCUSSION THREAD

**Discussion by MiguelRadaza** [livewire/livewire#5383] [7. Bugs (archived)]
There is an error when I use session()->flash(); 

Here is the error details:
![image](https://user-images.githubusercontent.com/48942632/204074650-7f24762b-a6af-4b2f-92a6-f712430dab88.png)
![image](https://user-images.githubusercontent.com/48942632/204074605-960508c2-f58d-4ba5-b1da-a0bb1bbcc790.png)

Here is my implementation:
![image](https://user-images.githubusercontent.com/48942632/204074687-885a2d15-6120-48b0-bd56-5ff437cb8847.png)

I'm using laravel 9 php 8.

**Comment by alfakiat** [comment]
function `deleteCourse` doesn't return any data.
Try to return data in JSON format.
and you can use [Postman](https://www.postman.com) to test your HTTP request.

**Comment by joshhanley** [comment]
@MiguelRadaza can you go to the network tab in dev tools in your browser, then refresh the page and then trigger your `deleteCourse` method.

There should now be an XHR (ajax) network request in the network tab, can you click on that, then go to the response tab and take a screenshot of what the server responded with?

It seems like it might have been an error somewhere causing a html response instead of json.

Thanks!
